lib/tasks/local_dev.rake

```diff
@@ -26,3 +24,18 @@ task setup_local_dev_data: :environment do
   end
   SupportUser.find_or_create_by!(dfe_sign_in_uid: 'dev-support', email_address: 'support@example.com')
 end
+
+desc 'Copy feature flags from production to your local dev env'
+task copy_feature_flags_from_production: :environment do
+  flags = JSON.parse(HTTP.get('https://www.apply-for-teacher-training.education.gov.uk/integrations/feature-flags'))
+
+  flags_to_set = flags['feature_flags'].reduce([]) do |acc, (flag_name, flag_data)|
+    acc.push(flag_name) if flag_data['active']
+    acc
+  end
+
+  FeatureFlag::FEATURES.each { |f| FeatureFlag.deactivate(f) }
+  (FeatureFlag::FEATURES & flags_to_set).each { |f| FeatureFlag.activate(f) }
```

Comment:
Perhaps to avoid the `flags_to_set`:

```suggestion
  FeatureFlag::FEATURES.each { |f| FeatureFlag.activate(f) if flags['features'][f]['active'] }
```

Reply:
better yet!